### DISCUSSION TITLE
How to stop strategy of camera

### DISCUSSION THREAD

**Discussion by berkayyildiz** [excaliburjs/Excalibur#2184] [Q&A]
Hello, I am creating a game that player create maps and play them. So I have play and create mods. When play mode active I want to use lockToActor camera strategy but when switched to create mode I want to stop that strategy because I let player move camera by pointer gestures in this mode. How can I stop the strategy?

**Comment by eonarheim** [comment]
Hi @berkayyildiz 

One way to stop the camera strategy from processing would be either call `camera.removeStrategy(...)` or `camera.clearAllStrategies()` and re-add to the strategy back to camera them when you switch back to the other mode.

Hope this helps!
-Erik
